File: Python/백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.py
aa = list(map(int,input().split()))
if(sum(aa)<100):
    if(min(aa)==aa[0]):
        print('Soongsil')
    elif(min(aa)==aa[1]):
        print('Korea')
    else:
        print('Hanyang')
else:
    print('OK')

File: C#/백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
            if(a.Sum()<100){
                if(a[0]==a.Min()){
                    Console.WriteLine("Soongsil");
                }
                else if(a[1]==a.Min()){
                    Console.WriteLine("Korea");
                }
                else{
                    Console.WriteLine("Hanyang");
                }
            }
            else{
                Console.WriteLine("OK");
            }
        }
    }
}